src/registry_schemas/schemas/agm_extension.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/agm_extension",
+    "type": "object",
+    "definitions": {},
+    "title": "AGM Extension Information Schema",
+    "required": [
+        "agmExtension"
+    ],
+    "properties": {
+        "agmExtension": {
+            "type": "object",
+            "required": [
+                "year",
+                "isFirstAgm"

```

Comment:
Is this required?
Is this something the user is sending us, or we're calculating and sending back?
would it be more helpful to have the date of the last agm? then the other business rules can be applied and a Null/"" value could be used to infer no prior agm?

Reply:
This is a required input from the user.

We do not necessarily have the date of the last AGM.

I figured we'd store this in the JSON so that the Legal API can validate the filing data (eg, if this is the first AGM then follow-on data is optional).

Reply:
Only year is now required. Thor, please resolve this conversation if OK with you.